File: unix/boot/spp/rpp/rppfor/puttok.f
      subroutine puttok (str)
      integer str (100)
      integer i
      i = 1
23000 if (.not.(str (i) .ne. -2))goto 23002
      call putchr (str (i))
23001 i = i + 1
      goto 23000
23002 continue
      return
      end

File: unix/boot/spp/rpp/rpprat/puttok.r
#-h-  puttok			  198  local   12/01/80  15:54:34
# puttok-put token into eval stack
   include  defs

    subroutine puttok (str)
    character str (MAXTOK)

    integer i

    for (i = 1; str (i) != EOS; i = i + 1)
       call putchr (str (i))
    return
    end
